server/app/templates/reset_password.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```

Comment:
These are the transactional email templates. Think this probably isn’t needed here?

Reply:
Ah, yeah. I guess that's not needed there then.